scripts/update-package.js

```diff
@@ -1,10 +1,10 @@
-const { writeFile, readdir } = require("fs/promises");
+import { writeFile, readdir, readFile } from "node:fs/promises"
 
 if (!process.env.OCTOKIT_OPENAPI_VERSION) {
   throw new Error("OCTOKIT_OPENAPI_VERSION is not set");
 }
 
-const pkg = require("../package.json");
+const pkg = JSON.parse((await readFile("../package.json")).toString());
```

Comment:
I don't think passing a relative path to `readFile` will work, did you try it out? For `readFile`, the passed path will be relative to `process.cwd()`, so I guess it should be

```suggestion
const pkg = JSON.parse((await readFile("package.json")).toString());
```

In order to read a file relative to the current path, you can use 

```js
import { dirname } from 'path';
import { fileURLToPath } from 'url';

const __dirname = dirname(fileURLToPath(import.meta.url));
```

[via stackoverflow](https://stackoverflow.com/a/50052194/206879)

Reply:
I have tested out the code. The relative path works just fine for me